audio_service_platform_interface/lib/method_channel_audio_service.dart

```diff
@@ -103,13 +106,13 @@ class MethodChannelAudioService extends AudioServicePlatform {
           print('### called callbacks.click');
           return null;
         case 'stop':
-          await callbacks.stop(StopRequest());
+          await callbacks.stop(const StopRequest());
```

Comment:
This will be inferred, and it may not always be `const` in the future. We can add parameters to the stop request in the future, so it may be more convenient to let inference do its thing.

Reply:
what you think about `const` constructors (with `@literal`), do they worth it? if you think it's better keep them out for more API extensibilty in future, i will remove them

Reply:
(as for messages in the interface)

Reply:
i don't think i understand you quite right. can you elaborate please? const is never inferred if you are about that

Reply:
Sorry it's been a while since I looked into it. Dart has some const inference but not in this context.

Reply:
OK here's the explanation of it: https://fuchsia.dev/fuchsia-src/development/languages/dart/style#dont_use_new_or_use_const_redundantly

> const can be inferred in:
> 
> * A const collection literal.
> * A const constructor call.
> * A metadata annotation.
> * The initializer for a const variable declaration.
> * A switch case expression—the part right after case before the :, not the body of the case.

Reply:
yes, it can be inferred from the outer `const` expression, but it's never applied automatically - that's what i meant